examples/plane/limiters_advection.jl

```diff
@@ -212,48 +215,49 @@ for (k, ne) in enumerate(ne_seq)
 
         # Compute hyperviscosity for the tracer equation by splitting it in two diffusion calls
         @. ystar.ρq = wdiv(grad(y.ρq / y.ρ))
-        Spaces.weighted_dss!(ystar)
+        Spaces.weighted_dss!(ystar.ρq)
         @. ystar.ρq = -D₄ * wdiv(y.ρ * grad(ystar.ρq))
 
         # Add advective flux divergence
-        @. ystar.ρ = -wdiv(y.ρ * u)         # contintuity equation
-        @. ystar.ρq += -wdiv(y.ρq * u)      # adevtion of tracers equation
-    end
+        @. ystar.ρ = y.ρ - dt * wdiv(y.ρ * u)         # contintuity equation
+        @. ystar.ρq += y.ρq - dt * wdiv(y.ρq * u)      # adevtion of tracers equation
+
 
-    function stage_callback!(ydoublestar, integrator, parameters, t)
-        space = parameters.space
         min_Q = parameters.min_Q
         max_Q = parameters.max_Q
 
         if lim_flag
             # Call quasimonotone limiter, to find optimal ρq (where ρq gets updated in place)
             Limiters.quasimonotone_limiter!(
-                ydoublestar.ρq,
-                ydoublestar.ρ,
+                ystar.ρq,
+                ystar.ρ,
                 min_Q,
                 max_Q,
                 rtol = limiter_tol,
             )
         end
-        Spaces.weighted_dss!(ydoublestar)
+        Spaces.weighted_dss!(ystar.ρ)
+        Spaces.weighted_dss!(ystar.ρq)
     end
 
+
     # Set up RHS function
     ystar = similar(y0)
     parameters = (space = space, min_Q = min_Q, max_Q = max_Q)
-    f!(ystar, y0, parameters, 0.0)
+    f!(ystar, y0, parameters, 0.0, dt)
 
     # Solve the ODE
-    prob = ODEProblem(f!, y0, (0.0, end_time), parameters)
-    sol = solve(
+    prob = ODEProblem(ForwardEulerODEFunction(f!), copy(y0), (0.0, end_time), parameters)
+    yend = solve(
```

Comment:
At the moment, TimeMachine doesn't store the full solution. We should probably have an option for that.

Reply:
Yes, that'd be great, so that we can do post-processing analysis easier rather than on-the-fly per each mesh refinement run